api/src/controllers/young/index.js

```diff
@@ -464,6 +464,35 @@ router.put("/", passport.authenticate("young", { session: false, failWithError:
   }
 });
 
+router.put("/:id/change-cohort/", passport.authenticate("young", { session: false, failWithError: true }), async (req, res) => {
+  try {
+    const { error, value } = validateYoung(req.body, req.user);
+    if (error) return res.status(400).send({ ok: false, code: ERRORS.INVALID_PARAMS, error: error.message });
+
+    const young = await YoungObject.findById(req.user._id);
+    if (req.body.isFull === true) {
+      young.set({ cohort: value.cohort, cohortChangeReason: value.cohortChangeReason, cohortDetailedChangeReason: value.cohortDetailedChangeReason, statusPhase1: "WAITING_LIST" });
```

Comment:
C'est le status global qu'il faut mettre en. waiting list... pas logique, je sais :(
```suggestion
      young.set({ cohort: value.cohort, cohortChangeReason: value.cohortChangeReason, cohortDetailedChangeReason: value.cohortDetailedChangeReason, status: "WAITING_LIST" });
```